src/Illuminate/Auth/Console/AuthMakeCommand.php

```diff
@@ -63,6 +64,28 @@ public function handle()
             );
         }
 
+        if (! $this->option('no-tests')) {
```

Comment:
Ok, so I extracted that into a method same as the `extractViews()` and it looks pretty good. Thanks 👍